docs/ops/movement/ScatterUpdate_3.md

```diff
@@ -7,49 +7,45 @@
 **Short description**: *ScatterUpdate* creates a copy of the first input tensor with updated elements specified with second and third input tensors.
 
 **Detailed description**: *ScatterUpdate* creates a copy of the first input tensor with updated elements in positions specified with `indices` input
-and values specified with `updates` tensor starting from the dimension with index `axis`. For the `data` tensor of shape `[d_0, d_1, ..., d_n]`,
-`indices` tensor of shape `[i_0, i_1, ..., i_k]` and `updates` tensor of shape
-`[d_0, d_1, ... d_(axis - 1), i_0, i_1, ..., i_k, d_(axis + 1), ..., d_n]` the operation computes
+and values specified with `updates` tensor starting from the dimension with index `axis`. For the `data` tensor of shape \f$[d_0, d_1, \dots, d_n]\f$,
+`indices` tensor of shape \f$[i_0, i_1, \dots, i_k]\f$ and `updates` tensor of shape
+\f$[d_0, d_1, \dots, d_{axis - 1}, i_0, i_1, \dots, i_k, d_{axis + 1}, \dots, d_n]\f$ the operation computes
```

Comment:
Could you attach a screenshot with rendered formulas after changes?

Reply:
![ScattrerUpdate](https://user-images.githubusercontent.com/72454773/130026693-2b2b8654-adb0-4650-919d-3aecdab2d817.png)

Reply:
Since whitespace characters are omitted inside $$, you may consider using a backslash `\` so they are rendered, like that:
```suggestion
\f$[d_0,\ d_1,\ \dots,\ d_{axis - 1},\ i_0,\ i_1,\ \dots,\ i_k,\ d_{axis + 1},\ \dots,\ d_n]\f$ the operation computes
```

Reply:
good catch, missed backslashes in this equation :)